agilex_piper/scene.xml

```diff
@@ -0,0 +1,21 @@
+<mujoco model="piper_scene">
+  <include file="piper.xml" />
+  <option integrator="implicitfast" impratio="1" />
+
+  <visual>
+    <headlight diffuse="0.6 0.6 0.6" ambient="0.1 0.1 0.1" specular="0 0 0" />
+    <global azimuth="120" elevation="-20" />
+  </visual>
+
+  <asset>
+    <texture name="grid" type="2d" builtin="checker" rgb1=".2 .3 .4" rgb2=".1 0.15 0.2" width="512" height="512"
+      mark="cross" markrgb=".8 .8 .8" />
+    <material name="grid" texture="grid" texrepeat="1 1" texuniform="true" />
+  </asset>
+
+  <worldbody>
+    <light pos="0 0 1.5" directional="true" />
+    <light pos="0 -1 0" directional="true" />
```

Comment:
If you want, you can remove one of the lights and add it to `piper.xml` and have it track the end-effector?

Reply:
e.g.

`<light name="spotlight" mode="targetbodycom" target="link8" pos="0 -1 2"/>`